postgresql/init.sls

```diff
@@ -3,3 +3,6 @@ include:
 {% if pillar.postgresql.server is defined %}
 - postgresql.server
 {% endif %}
+{% if pillar.postgresql.cluster is defined %}
```

Comment:
If you mean something like postgresql:server:cluster I didn't see this structure in any Salt formula in this namespace that already supports both single and cluster deployments. For example RabbitMQ and Redis also have rabbitmq:cluster and rabbitmq:server, redis:cluster and redis:server metadata